chain/rosetta-rpc/src/models.rs

```diff
@@ -1301,3 +1326,91 @@ impl From<SignatureType> for near_crypto::KeyType {
         }
     }
 }
+
+// *** NEP-141 FT ***
+#[derive(serde::Serialize, serde::Deserialize, Debug)]
+pub(crate) struct Nep141Event {
+    pub version: String,
+    #[serde(flatten)]
+    pub event_kind: Nep141EventKind,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Debug)]
+#[serde(tag = "event", content = "data")]
+#[serde(rename_all = "snake_case")]
+#[allow(clippy::enum_variant_names)]
+pub(crate) enum Nep141EventKind {
+    FtTransfer(Vec<FtTransferData>),
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Debug)]
+pub(crate) struct FtMintData {
+    pub owner_id: String,
+    pub amount: String,
+    pub memo: Option<String>,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Debug)]
+pub(crate) struct FtTransferData {
+    pub old_owner_id: String,
+    pub new_owner_id: String,
+    pub amount: String,
+    pub memo: Option<String>,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Debug)]
+pub(crate) struct FtBurnData {
+    pub owner_id: String,
+    pub amount: String,
+    pub memo: Option<String>,
+}
+#[derive(Debug, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct FungibleTokenEvent {
+    pub standard: String,
+    pub receipt_id: CryptoHash,
+    pub block_height: u64,
+    pub block_timestamp: u64,
+    pub contract_account_id: String,
+    pub symbol: String,
+    pub decimals: u32,
+    pub affected_account_id: String,
+    pub involved_account_id: Option<String>,
+    pub delta_amount: i64,
```

Comment:
I believe an `i64` is too small? According to NEP-141, the amount is stored in `u128`, right? So delta could be up to  +/- `u128`, which means not even `i128` is large enough :(